src/docs/spell.ok

```diff
@@ -5,9 +5,13 @@ ActiveState
 Adler's
 ASAN
 Atomicity
+autconf
```

Comment:
Is that just a typo?

Reply:
Nice find, that is a typo (kind've embarrassing I almost added a typo to the spell checker). I've updated to docs with the correct spelling :)